backends/rapidpro/channel.go

```diff
@@ -353,12 +349,7 @@ func (c *DBChannel) HasRole(role courier.ChannelRole) bool {
 
 // ConfigForKey returns the config value for the passed in key, or defaultValue if it isn't found
 func (c *DBChannel) ConfigForKey(key string, defaultValue interface{}) interface{} {
-	// no value, return our default value
-	if !c.Config_.Valid {
```

Comment:
we should convert fields like Channel.config and Org.config to be actual JSON so there's no danger of them becoming invalid